examples/DotNetCore/ConsoleApplication/Program.cs

```diff
@@ -0,0 +1,94 @@
+﻿namespace Microsoft.Extensions.Configuration.AzureAppConfiguration.Examples.ConsoleApplication
+{
+    using Microsoft.Extensions.Configuration;
+    using Microsoft.Extensions.Configuration.AzureAppConfiguration;
+    using System;
+    using System.Text;
+    using System.Threading;
+    using System.Threading.Tasks;
+
+    class Program
+    {
+        static IConfiguration Configuration { get; set; }
+        static IConfigurationRefresher _refresher;
+
+        static void Main(string[] args)
+        {
+            Configure();
+
+            var cts = new CancellationTokenSource();
+            Run(cts.Token);
+
+            // Finish on key press
+            Console.ReadKey();
+            cts.Cancel();
+        }
+
+        private static void Configure()
+        {
+            var builder = new ConfigurationBuilder();
+
+            // Load a subset of the application's configuration from a json file and environment variables
+            builder.AddJsonFile("appsettings.json")
+                   .AddEnvironmentVariables();
+
+            IConfiguration configuration = builder.Build();
+
+            if (string.IsNullOrEmpty(configuration["connection_string"]))
```

Comment:
It will be nice to have some consistency of the environment variable name of the connection string (so make it easy for someone to try out different examples). Do you like to change this one to `ConnectionString` like the one you are using in the webjob example?

Reply:
Removed it.